### PR TITLE
Add description: only support SCSI for hotplug volume

### PR DESCRIPTION
**Pull request by futuretea**
Signed-off-by: futuretea <Hang.Yu@suse.com>

**Related issues**
https://github.com/harvester/harvester/issues/2501

### CODE REVIEW

File: docs/vm/hotplug-volume.md [hunk 1]
```diff
@@ -11,6 +11,12 @@ Description: Adding hot-plug volumes to a running VM.
 
 Harvester supports adding hot-plug volumes to a running VM.
 
+:::info
+
+Currently, only support disk bus `scsi` for hot-plug volumes, refer to this [description](https://github.com/kubevirt/kubevirt/issues/5080#issuecomment-785183128)
```
Comment (vickyhella) [inline docs/vm/hotplug-volume.md:16]:
```suggestion
Currently, KubeVirt only supports disk bus `scsi` for hot-plug volumes. For more information, see this [issue](https://github.com/kubevirt/kubevirt/issues/5080#issuecomment-785183128).
```

**Review by vickyhella — COMMENTED** [review]
Just some small suggestion

**Review by vickyhella — APPROVED** [review]

**Review by guangbochen — COMMENTED** [review]
also need to backport to the v1.1 doc, thanks.

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Pull request merged**